readme.md

```diff
@@ -162,6 +162,7 @@ A variety of different types of results can be extracted from the odbs and compa
 - `slope`: finds the slope of an xy data set
 - `finalValue`: finds the y value at the last increment in the xy data set
 - `x_at_peak_in_xy`: finds the x-value corresponding to the absolute peak in the y-value
+- `tabular`: given a list of tuples [(x,y)...] confirms that a corresponding point exists in requested output
```

Comment:
Use same description as in processresults.py header

Reply:
One more note: we are trying to adhere to the PEP-8 style. I didn't check to see this code is in compliance with PEP-8.